### PR TITLE
issue #153: donate button

### PR DESCRIPTION
**Pull request by fkorning**
please ignore false file permissions (weird side-effect of mounting git repo via vm).
also ignore Gemfile (had to tweak it cause I'm on ruby-2.6.5 compiled from source).

### CODE REVIEW

**Comment by fkorning** [comment]
https://user-images.githubusercontent.com/8026632/69922782-24e7a700-1497-11ea-9ab5-bd4d58114959.png

**Comment by tdg** [comment]
The button looks good. I'd use the same magenta as the logo, and spell with a capital D - Donate (and no £ sign)

@aspiers @jpallen I'm not sure how to request a review so here I go!

**Comment by tdg** [comment]
@aspiers @jpallen I'm not sure how to request a review so here I go!

**Comment by fkorning** [comment]
Are you sure you don't want the GBP icon?  also, I can modify the login to make it a button as well like this:
image

**Comment by fkorning** [comment]
with active hover both buttons highlight to the $button-color (blue).

image

**Comment by fkorning** [comment]
Capitalised, no icons.
image

File: app/assets/stylesheets/titler.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    .list-group-item {
+    }
+   
+    .button {
+        background-color: $faded-color;
+        color: $white;  
+        font-weight: normal;
+        padding: 0px 4px 0 4px;
```
[the remark this answers is not in the document]
Comment (fkorning) [reply app/assets/stylesheets/titler.scss]: quixotic much?

**Comment by fkorning** [comment]
@tdg:  
earlier you suggested "I'd use a different colour from the logo and the main call to action - probably a green or possibly grey"

but then you also suggested "The button looks good. I'd use the same magenta as the logo, and spell with a capital D - Donate (and no £ sign)"

which one would you like: magenta or grey ?

**Comment by fkorning** [comment]
capitalised greyed icons

image

**Comment by aspiers** [comment]
@fkorning Thanks for your work so far.  This PR has a lot of unnecessary changes on it (including changing the permissions of every single file in the repo) which need to be removed before we can accept it.  I've attempted to clean up the branch, rebased, and pushed to my `donate` branch so you could reset your branch to that if it helps.

**Comment by fkorning** [comment]
@aspiers about the permissions...
yes... I had the git repo mounted in the vm via a shared folder. 
I'll do it again in a freh repo built entirely within the guest vm..

**Comment by aspiers** [comment]
Thanks!

**Comment by fkorning** [comment]
Killing this repo on account of exec permissions set by mounting repo via virtualbox shared folders.
Redoing this with a clean new repo cloned into the vagrant-ubuntu-eoan-swapmyvote vm filesystem.

**Pull request closed without merging**